src/content/docs/guides/setting-up-cloudwatch-dashboards.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Setting up CloudWatch Dashboards
+description: How to set up a basic CloudWatch Dashboard
+---
+
```

Comment:
This could probably be better with screenshots, but I don't want to expose any of our logs, and I think they'd be out of date pretty fast. The AWS docs linked have a good amount of screenshots.